.github/workflows/ci.yml

```diff
@@ -1510,6 +1514,9 @@ jobs:
             xtensa-espressif_esp32s2_zephyr-elf)
               PLATFORM_ARGS+="-p esp32s2_saola "
               ;;
+            xtensa-espressif_esp32s3_zephyr-elf)
+              PLATFORM_ARGS+="-p esp32s3_devkitc "
```

Comment:
`esp32s3_devkitc` board does not exist in the upstream Zephyr and this test will not pass.

SDK CI passes now only because it runs as `pull_request_target` and uses the CI workflow files from the base branch.

Reply:
@stephanosio, you are correct. I submitted this PR so that I can retrieve the toolchain and test with `esp32s3_devkitc` that is currently being ported. Thanks for merging into `topic-esp32s3`.